client/src/app/components/contact-us/contact-us.component.scss

```diff
@@ -0,0 +1,28 @@
+@import "../../../assets/vars.scss";
+
+.op-contact-us {
+
+  h2{
+    color: #4B4B4B;
+    font-size: 24px;
+    font-weight: bolder;
+  }
+
+  input, textarea {
+    border-radius: 3px;
+    border-width: 1px;
+    border-color: #B1B1B1;
+    width:500px;
+
+  }
+  color: #A3A3A3;
```

Comment:
Move those rules right under the `.op-contact-us` selector (before the  sub-selectors).